src/CHIPBackend.cc

```diff
@@ -1346,6 +1350,9 @@ size_t CHIPBackend::getNumDevices() {
   return NumDevices;
 }
 std::vector<std::string *> &CHIPBackend::getModulesStr() { return ModulesStr_; }
+const std::vector<std::string *> &CHIPBackend::getModulesStr() const {
+  return ModulesStr_;
+}
```

Comment:
Seems unnecessary?

Reply:
It's for calling the method via `const` qualified pointers and references.

Reply:
I meant that why have two of these? Why not get rid of the old non-const function and create a `addModulesStr()` and `removeModulesStr()` 

having a const and a non-const version of the same thing seems a bit redundant/confusing.

Reply:
Removed the getModuleStr.